interfaces/builtin/physical_memory_observe.go

```diff
@@ -0,0 +1,41 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const physicalMemoryObserveConnectedPlugAppArmor = `
+# Description: With kernels with STRICT_DEVMEM=n, read-only access to all physical
+# memory. With STRICT_DEVMEM=y, allow reading /dev/mem for read-only
```

Comment:
@morphis - I'm not aware of a way to check for STRICT_DEVMEM at runtime but the snappy kernel porting guide states that STRICT_DEVMEM must be used to be a proper snappy kernel. If someone turns it off, my feeling is that the interface code shouldn't be the one verifying that anyway. I *think* this should be handled via a verification tool (perhaps the kernel team has thought through this?).